molgenis-data-security/src/main/java/org/molgenis/data/security/EntityTypePermission.java

```diff
@@ -16,7 +16,9 @@
 	UPDATE_DATA("Permission to update entities of this EntityType"),
 	DELETE_DATA("Permission to delete entities of this EntityType"),
 	UPDATE_METADATA("Permission to update the metadata of this EntityType"),
-	DELETE_METADATA("Permission to delete the data and metadata of this EntityType");
+	DELETE_METADATA("Permission to delete the data and metadata of this EntityType"),
+	MANAGE_REPORT("Permission to manage the report for this EntityType"),
```

Comment:
Reports are `molgenis-dataexplorer` specific, the data-security module shouldn't have any knowledge of this.

Reply:
Discussed with @sidohaakma: will look into another solution